include/SDL3/SDL_stdinc.h

```diff
@@ -49,11 +49,15 @@
 #include <SDL3/SDL_platform_defines.h>
 
 #include <stdarg.h>
+#if defined (HAVE_STDINT_H)
 #include <stdint.h>
+#endif
 #include <string.h>
+#if defined (HAVE_WCHAR_H)
 #include <wchar.h>
```

Comment:
On most platforms, the SDL3 headers and implementation have a hard dependency on `stdint.h` and `wchar.h`.
So you should add a minimum implementation to the ngage toolchain.

In SDL3, the headers should be usable without various `HAVE_xxx` macros.

Reply:
Thanks, I will look into it ASAP.

Reply:
Done.